src/util/graph.h

```diff
@@ -458,6 +458,34 @@ void grapht<N>::visit_reachable(node_indext src)
     nodes[index].visited = true;
 }
 
+/// Add to `set` nodes that are reachable from `set`
+/// \param set: set of source nodes, must be a container with an
+///   `insert(const value_type&)` method.
+/// \param for_each_successor: function which given a node `n` and a function
+///   `f`, applies `f` on all successors of `n`.
+template <class Container, typename nodet = typename Container::value_type>
+void get_reachable(
```

Comment:
This function is introduced, but never used in this PR?

Reply:
It used in https://github.com/diffblue/cbmc/pull/1947/commits/b2ca93723b89c236b8ceb5be3caa2266e60bb998